src/main/webapp/js/common/react/tests/Navbar.test.js

```diff
@@ -2,59 +2,86 @@ import React from 'react';
 import Navbar from '../components/Navbar.js';
 import '@testing-library/jest-dom/extend-expect';
 import {render, screen} from '@testing-library/react';
-import fetchMock from 'fetch-mock-jest';
-import {makeRelativeUrlAbsolute} from '../../../fetch_handler.js';
 
 const navbarUrls = [
   {
     href: '/',
     name: 'Home',
+    requiresAuth: false,
   },
   {
     href: '/projects.html',
     name: 'Projects',
+    requiresAuth: false,
+  },
+  {
+    href: '/dashboard.html',
+    name: 'Dashboard',
+    requiresAuth: true,
   },
 ];
 
-beforeAll(() => {
-  const authData = {
-    authorized: false,
-    loginUrl: '/login',
-    logoutUrl: '/logout',
-    hasProfile: false,
-  };
-
-  fetchMock.get(makeRelativeUrlAbsolute('/auth'), {
-    body: JSON.stringify(authData),
-    status: 200,
-    headers: {
-      'Content-Type': 'application/json',
-    },
-  });
-});
-
-afterAll(() => {
-  fetchMock.restore();
-});
+const mockAuthData = {
```

Comment:
Naming is slightly confusing here because you define other mock auth data later that you just call 'authData.'
Maybe if you can imply with the name that this is the authdata for an authorizd user with a profile, or even just 'defaultAuthData'?

Reply:
That's a good point! Will update, thanks!